docs/docs/.vuepress/components/PythonMappings.vue

```diff
@@ -0,0 +1,193 @@
+<style lang="stylus"></style>
+
+<template>
+  <div class="md:w-3/4 m-auto">
+    <div
+      class="
+        bg-gray-100
+        py-8
+        px-4
+        w-full
+        border-l-8 border-green-500 border-solid
+      "
+    >
+      <h3 class="text-bold text-lg text-green-500 mb-2">
+        Don't see a package listed?
+      </h3>
+      Request to have a custom package mapping added:
+      <form
+        class="grid grid-cols-12"
+        v-on:submit.prevent="submitPackageRequest"
+      >
+        <input
+          v-model="requestedPackage"
+          placeholder="PyPi package name"
+          class="
+            my-3
+            px-3
+            py-3
+            placeholder-slate-300
+            text-slate-600
+            relative
+            bg-white
+            rounded-l
+            text-sm
+            border-0
+            shadow
+            outline-none
+            focus:outline-none focus:ring
+            w-full
+            m-auto
+            col-span-10
+          "
+        />
+        <button
+          :disabled="!requestedPackage"
+          class="
+            bg-green-500
+            border-green-500
+            text-white
+            font-bold
+            border-4
+            py-3
+            my-3
+            col-span-2
+            rounded-r
+            disabled:opacity-50
+          "
+        >
+          Submit
+        </button>
+        <span
+          v-if="successMessage"
+          class="text-green-500 font-semibold col-span-5 italic"
+        >
+          {{ successMessage }}
+        </span>
+        <span
+          v-if="errorMessage"
+          class="text-red-500 font-semibold col-span-5 italic"
+        >
+          {{ errorMessage }}
+        </span>
+      </form>
+    </div>
+    <div class="m-auto my-8">
+      <h2 class="text-xl">Search PyPi Mappings</h2>
```

Comment:
This is cool but I have a couple of concerns:

* It reads like this is a searchable list of _all_ packages, but `pipreqs` actually implements this logic: 1) search for the `import` name on PyPI, and if a package with that name doesn't exist, 2) look up the `import` name in the mapping file to see if a mapping exists. So this isn't a full list of packages.
* Do you have a plan for keeping this list up-to-date with the list in the EE in prod?

Reply:
It might be nice to note to the user that all they have to do is `import package` and it should _just work_ (the user doesn't really need to know whether `pipreqs` finds the package or has to look it up from the mapping file). 

Then we can tell the user: if `import package` doesn't work, they can fill out the request for us to add support for the package. But we technically don't need to tell them that there's a mapping file involved at all. It's not a secret, but I think it overcomplicates the explanation, since we'll still tell them to `import package` once we add the mapping anyway. wdyt?